### ISSUE TITLE
Inconsistent color contrast calculation

### ISSUE THREAD

**Issue by DhirajKhot** [component-driven/cypress-axe#98]
The color contrast ratio differs everytime upon running the scan and sometimes reports the violated result for the same styled element.

Below is the snapshot of the computed style of the element for which issue is being reported -
![image](https://user-images.githubusercontent.com/20378231/117964203-f6822280-b33e-11eb-9e04-3bf9edf6d9c0.png)

For the same element we get violations with different color contrast ratios - 
![image](https://user-images.githubusercontent.com/20378231/117963562-2aa91380-b33e-11eb-9a0b-b232ddceefaa.png)
![image](https://user-images.githubusercontent.com/20378231/117963494-15cc8000-b33e-11eb-81dd-c2d814486498.png)
![image](https://user-images.githubusercontent.com/20378231/117963602-3a285c80-b33e-11eb-8ab4-d76e76cee2eb.png)

Any idea what could be the issue here?

**Comment by stereobooster** [comment]
I guess something of this https://github.com/dequelabs/axe-core/issues?q=is%3Aissue+is%3Aopen+contrast+label%3A%22color+contrast%22

**Comment by DanielleCogs** [comment]
Any updates on this? We are still experiencing this issue
